### DISCUSSION TITLE
Is there exist correlation between gene family expansion and gene copy number increase?

### DISCUSSION THREAD

**Discussion by wjl-cpu** [hahnlab/CAFE5#163] [Q&A] [answered]
Hello,author.
I am currently investigating the adaptation of a species to the marine environment. Initially, I utilized CAFE5 software to acquire the gene families that underwent expansion and contraction in this  species. Subsequently, an enrichment analysis was conducted on the expanding gene families. I identified genes that were significantly enriched on one of these pathways and counted the number of copies of these genes in both the species and its relatives (non-marine species). The number of copies of these genes was not found to increase in this species compared to other species.
I would like to know why this result occurs, and whether there is no correlation between the expansion/contraction of gene families and the increase/decrease of gene copy number. Could you give me some advice?
Best wishes for you!

**Comment by hahnlab-user** [comment, marked as answer]
Yes, gene family expansions should always be accompanied by increases in gene copy number--mostly because they are the same thing! Maybe I am not understanding how "gene copies" differs here.
